README.md

```diff
@@ -201,184 +108,53 @@ You'll need to have a Spark release on your system and the `$SPARK_HOME` environ
-time than samtools to compute these statistics. For example, on a MacBook Pro
-`flagstat NA12878_chr20.bam` took 17 seconds to run while `samtools flagstat NA12878_chr20.bam`
-took 55 seconds. On larger files, the difference in speed is even more dramatic. ADAM is faster
-because it's multi-threaded and distributed and uses a columnar storage format (with a
-projected schema that only materializes the read flags instead of the whole read). 
-
-### `adam-shell`
-
-The `adam-shell` command opens an interpreter that you can run ad-hoc ADAM commands in.
-
-For example, the following code snippet will generate a result similar to [the k-mer-counting example above](./README.md#hello-world-counting-k-mers), but with the k-mers sorted in descending order of their number of occurrences. To use this, save the code snippet as `kmer.scala` and run `adam-shell -i kmer.scala`.
-
-`kmer.scala`
-```scala
-import org.bdgenomics.adam.rdd.ADAMContext._
-import org.bdgenomics.adam.projections.{ AlignmentRecordField, Projection }
-
-// Load alignments from disk
-val reads = sc.loadAlignments(
-  "/data/NA21144.chrom11.ILLUMINA.adam",
-  projection = Some(
-    Projection(
-      AlignmentRecordField.sequence,
-      AlignmentRecordField.readMapped,
-      AlignmentRecordField.mapq
-    )
-  )
-)
-
-// Generate, count and sort 21-mers
-val kmers =
-  reads
-    .rdd
-    .flatMap(_.getSequence.sliding(21).map(k => (k, 1L)))
-    .reduceByKey(_ + _)
-    .map(_.swap)
-    .sortByKey(ascending = false)
-
-// Print the top 10 most common 21-mers
-kmers.take(10).foreach(println)
-```
-`adam-shell -i kmer.scala`
-```
-$ adam-shell -i kmer.scala
-…
-(121771,TTTTTTTTTTTTTTTTTTTTT)
-(44317,ACACACACACACACACACACA)
-(44023,TGTGTGTGTGTGTGTGTGTGT)
-(42474,CACACACACACACACACACAC)
-(42095,GTGTGTGTGTGTGTGTGTGTG)
-(33797,TAATCCCAGCACTTTGGGAGG)
-(33081,AATCCCAGCACTTTGGGAGGC)
-(32775,TGTAATCCCAGCACTTTGGGA)
-(32484,CCTCCCAAAGTGCTGGGATTA)
-…
-```
-
-# Running on a cluster
-
-The `adam-submit` and `adam-shell` commands can also
-be used to submit ADAM jobs to a Spark cluster, or to run ADAM interactively. Cluster mode can be enabled by passing [the same flags you'd pass to Spark](https://spark.apache.org/docs/1.6.3/submitting-applications.html#launching-applications-with-spark-submit), e.g. `--master yarn --deploy-mode client`.
-
-# Under the Hood
-ADAM relies on several open-source technologies to make genomic analyses fast and massively parallelizable…
-
-## Apache Spark
-
-[Apache Spark][Spark] allows developers to write algorithms in succinct code that can run fast locally, on an in-house cluster or on Amazon, Google or Microsoft clouds. 
-
-## Apache Parquet
+# Documentation
 
-[Apache Parquet][Parquet] is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.
+(index from readthedocs.io)
+(include link to API docs at http://bdgenomics.org/adam/latest/scaladocs/index.html)
```

Comment:
Sorry, the links here still need to be fleshed out, I assume the main doc link would be to http://adam.readthedocs.io/en/latest/

Reply:
LOL whoops. I'll clean these for you.